resources/views/layouts/input.blade.php

```diff
@@ -1,5 +1,5 @@
 <div class="field">
 <div class="control">
-<input class="input" type="text" placeholder="Text input">
+<input class="input is-rounded" type="text" placeholder="Rounded input">
```

Comment:
I've got some advice :smile:  Change (temporarily) your input.blade.php html-file - just select html as your language in vscode - instead of php. Then right click and press _format document_